docs/en/styling-and-theming/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+As a HTML-based technology, Dojo makes use of CSS for styling elements across the framework and for applications developed with it.
+
+Dojo promotes encapsulated structural styling of individual widgets for maximum reuse, as well as simplified presentational theming across all widgets within an application. This pattern gives users a predictable way to style and theme their applications, even when using a mixture of widgets from [Dojo's `@dojo/widgets` library](https://github.com/dojo/widgets), a third-party provider, or any that may be developed in-house for a particular application.
+
+| Feature                               | Description                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
The content in this table seems more like a series of concepts than tabular data, and might be better presented as a series of headings with paragraphs.

Reply:
This is just the introduction section though, and follows the convention established in the `i18n` guide. It's only really intended as an elevator pitch, highlighting the main features/selling points of the capability - ideally as one-liner descriptions.

I think the ideal presentation for these is in a description list, but markdown doesn't easily support those without explicit HTML tags unfortunately! Tables felt like the best compromise given all the above.